cachito/web/metrics.py

```diff
@@ -0,0 +1,28 @@
+import os
+import socket
+
+from prometheus_client import multiprocess
+from prometheus_client.core import CollectorRegistry
+from prometheus_flask_exporter.multiprocess import GunicornInternalPrometheusMetrics
+
+hostname = socket.gethostname()
```

Comment:
Could this be called inside the `init_metrics()` function? It probably doesn't matter much, just feels cleaner.

Reply:
Moved.  I was considering it a constant that I'd never change, hence putting it up top.  On the flip side, its only used in that one place.